README.md

```diff
@@ -10,16 +10,37 @@ alpinejs-devtools is a simple extension to help you debug [Alpine.js](https://gi
 
 ## Installation
 
-- [Get the Chrome Extension](https://chrome.google.com/webstore/detail/alpinejs-devtools/fopaemeedckajflibkpifppcankfmbhk)
-- [Get the Firefox Extension](https://addons.mozilla.org/firefox/addon/alpinejs-devtools/)
+-   [Get the Chrome Extension](https://chrome.google.com/webstore/detail/alpinejs-devtools/fopaemeedckajflibkpifppcankfmbhk)
+-   [Get the Firefox Extension](https://addons.mozilla.org/firefox/addon/alpinejs-devtools/)
+
+If you are using Google Chrome, make sure that you are have gave the extension a permission to `Access to File URLs`, you can do this by following these steps:
```

Comment:
If you are using Google Chrome, Alpine Devtools won't work with local files unless you configure the extension to be allowed `Access to File URLs`